### PR TITLE
[AF-1904]: Changed simple for filename analyzer

### PR DESCRIPTION
**Pull request by adrielparedes**
# Description

Upgrading ISPN to last version because it contains some new analyzers we need. Also changed **simple** for **filename** analyzer.

@ederign @porcelli @tomasdavidorg 

# Related to

* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/938
* https://github.com/kiegroup/appformer/pull/663
* https://github.com/kiegroup/kie-wb-common/pull/2531

### CODE REVIEW

**Comment by adrielparedes** [comment]
[Upgrade - https://issues.jboss.org/browse/AF-1904](https://trello.com/c/1oeK5qMo/12-upgrade-https-issuesjbossorg-browse-af-1904)

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by ederign** [comment]
please jenkins execute full downstream build

File: uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/infinispan/suite/InfinispanTestSuite.java [hunk 1]
```diff
@@ -54,7 +54,7 @@
 public class InfinispanTestSuite {
 
     @ClassRule
-    public static ContainerDslRule infinispan = new ContainerDslRule("jboss/infinispan-server:9.3.2.Final")
+    public static ContainerDslRule infinispan = new ContainerDslRule("jboss/infinispan-server:9.4.9.Final")
```
Comment (mareknovotny) [inline uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/infinispan/suite/InfinispanTestSuite.java]: could we have this filtered by maven from https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/841856b8a8de72586d718eae2bd8362fd91adaad/pom.xml#L243 version property?

Comment (ederign) [reply uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/infinispan/suite/InfinispanTestSuite.java]: +1

Comment (adrielparedes) [reply uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/infinispan/suite/InfinispanTestSuite.java]: @mareknovotny but is this going to work if someone wants to run this test alone without maven, or from IDE?

Comment (mareknovotny) [reply uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/infinispan/suite/InfinispanTestSuite.java]: @adrielparedes it won't work without Maven or support of that in IDE. I know m2eclipse does that for Eclipse and filtering worked for me.
I am not sure about Intellij Idea, but I guess it supports it in some way too even you can run a test through Maven goal execution I think.

Comment (porcelli) [reply uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/infinispan/suite/InfinispanTestSuite.java]: if for any reason, the IDE doesn't work @adrielparedes, I'd prefer to have someone touching it manually local to make it work.. instead to have the version fixed in the code.

+1 for @mareknovotny

Comment (adrielparedes) [reply uberfire-extensions/uberfire-metadata/uberfire-metadata-commons-io/src/test/java/org/uberfire/ext/metadata/io/infinispan/suite/InfinispanTestSuite.java]: @porcelli @mareknovotny great I will update the PR today. Thanks guys

**Comment by adrielparedes** [comment]
@porcelli @mareknovotny @ederign done. Also updated some tests to take less time

**Comment by adrielparedes** [comment]
@tomasdavidorg on monday we should retest together this (I already did a smoke tests and worked fine) with those selenium tests, and if everything works fine is ready.

**Comment by mareknovotny** [comment]
@adrielparedes awesome, thanks for the update!

**Pull request merged**